### PR TITLE
Update GEL Typography values

### LINKED ISSUE #565: Update GEL typograhy values
**Is your feature request related to a problem? Please describe.**
We have to update the Typography values to match the latest changes in the UX Font Stacks & Styles spreadsheet.

**Describe the solution you'd like**
Go through all the scripts files and update the values that have changed.

For example:

Update `latin-and-cyrillic` typography `groupA` values as follows:

doublePica: {
    groupA: {
      fontSize: ~~20~~ **18**,
      lineHeight: ~~24~~ **22**,
},

brevier: {
    groupA: {
      fontSize: 14,
      lineHeight: ~~16~~ **18**,
},

**Describe alternatives you've considered**
N/A

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

**Additional context**

### PR DESCRIPTION
**Pull request by DenisHdz**
Resolves #565

**Overall change:** 
Update the Typography values to match with the latest changes in the UX Font Stacks & Styles spreadsheet. 

**Code changes:**
- Go through all the scripts files and update the values that have changed.
- Update `getTypeSizes` test

---

- [X] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

**Review by dr3 — CHANGES_REQUESTED** [review]
Can this be a breaking change? Dont want to pull it in by mistake,

Other than that LGTM so long as the UX review is very intensive to ensure the values are correct

**Review by ghost — DISMISSED** [review]

File: packages/utilities/gel-foundations/CHANGELOG.md [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
-| 0.2.4   | [PR#323](https://github.com/BBC/psammead/pull/323) Update readme storybook badge |
-| 0.2.3   | [PR#264](https://github.com/BBC/psammead/pull/319) Resolving package-lock issues. |
-| 0.2.2   | [PR#304](https://github.com/BBC-News/psammead/pull/304) Add font family definitions, which include GEL-defined font fallbacks. |
-| 0.2.1   | [PR#303](https://github.com/BBC-News/psammead/pull/303) Bump to ensure correct build is published |
-| 0.2.0   | [PR#290](https://github.com/BBC-News/psammead/pull/290) Move typography font-sizes to use rem instead of em |
-| 0.1.4   | [PR#224](https://github.com/BBC-News/psammead/pull/224) Add tests for the exported values, coverage 100% :tada: |
-| 0.1.1   | [PR#212](https://github.com/BBC-News/psammead/pull/212) Update package description and README. |
+| Version | Description                                                                                                                                                  |
+| ------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------ |
```
Comment (dr3) [inline packages/utilities/gel-foundations/CHANGELOG.md]: Could you add the prettier ignore comment here and reset the styling please?

**Review by ghost — APPROVED** [review]

**Review by dr3 — APPROVED** [review]

**Comment by WeimingFan** [comment]
This Typography values is approved against UX GEL setting.  Fan

**Comment by WeimingFan** [comment]
This Typography values is approved against UX GEL setting.  Fan

**Comment by jamesbrumpton** [comment]
LGTM.

**Pull request merged**